src/components/datatable/DataTable.js

```diff
@@ -4,13 +4,17 @@ import paginationFactory from 'react-bootstrap-table2-paginator';
 import { Modal, Button } from 'react-bootstrap';
 import UserModal from '../userModal/UserModal';
 import './DataTable.scss';
+import 'react-bootstrap-table-next/dist/react-bootstrap-table2.min.css';
+import 'react-bootstrap-table2-paginator/dist/react-bootstrap-table2-paginator.min.css';
+import 'react-bootstrap-table2-filter/dist/react-bootstrap-table2-filter.min.css';
+
 
 class DataTable extends Component {
```

Comment:
Since we are updating this component, can we convert it to a functional component + hooks ? The idea is to gradually get rid of all class components !